src/fieldAnalyser/fieldsUtils.spec.ts

```diff
@@ -1,7 +1,10 @@
 jest.mock('@coveo/platform-client');
 
-import PlatformClient, {FieldModel} from '@coveo/platform-client';
-import {createFields, listAllFieldsFromOrg} from './fieldUtils';
+import PlatformClient, {FieldModel, FieldTypes} from '@coveo/platform-client';
+import {Inconsistencies} from './inconsistencies';
+import {FieldTypeInconsistencyError} from '../errors';
+import * as Utils from './fieldUtils';
+const {createFields, listAllFieldsFromOrg, createFieldsFromReport} = Utils;
```

Comment:
:suspect: ? why that

Reply:
to put a spy on the `createFields` method (making sure it is being called). Since this is the module I am testing, it would not make sense to mock it.